### DISCUSSION TITLE
PIMCORE Custom Reports

### DISCUSSION THREAD

**Discussion by KasthuriAnandhan** [pimcore/pimcore#17203] [Q&A / Support]
Hi Team,
@kingjia90 

I'm trying to install pimcore's custom report bundle, but I can't able to find any document to install it even in pimcore documentation also, anybody can help me to this.

Thanks,
Kasthuri Anandhan

**Comment by kingjia90** [comment]
Hi,

Yes, it's an optional bundle and it's in the core but needs to be installed as instructed on the "⚠️Caution" part on top of the page in
https://pimcore.com/docs/platform/Pimcore/Tools_and_Features/Custom_Reports/

Let me know, Best regards

**Comment by KasthuriAnandhan** [comment]
Hi @kingjia90  I was tried as they said in the documentation,

<?php
use Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle;
use Pimcore\Bundle\CustomReportsBundle\PimcoreCustomReportsBundle;


return [
    \Pimcore\Bundle\EnterpriseSubscriptionToolsBundle\PimcoreEnterpriseSubscriptionToolsBundle::class => ['all' => true],
    \Pimcore\Bundle\PimcoreCustomReportsBundle\PimcorePimcoreCustomReportsBundle::class => ['all' => true],
];

When I save the bundles.php file I facing the following error

Attempted to load class "PimcoreCustomReportsBundle" from namespace "Pimcore\Bundle\CustomReportsBundle".
Did you forget a "use" statement for another namespace?

**Reply by kingjia90** [reply]
`Pimcore\Bundle\PimcoreCustomReportsBundle\PimcorePimcoreCustomReportsBundle`... there is a little too much Pimcore in there
it's  `Pimcore\Bundle\CustomReportsBundle\PimcoreCustomReportsBundle`

see also https://github.com/pimcore/demo/blob/8bcf5b4906cfcf65c832462532cd10a1d7af2433/config/bundles.php#L20

**Comment by KasthuriAnandhan** [comment]
@kingjia90 could you please help on this, I was trying this so far.

**Comment by betterapp** [comment]
you need to add this bundle i bundles.php and then clear cache ./bin/console cache:clear and then install bundle
